src/ddmd/parse.d

```diff
@@ -21,6 +21,7 @@ import ddmd.root.filename;
 import ddmd.root.outbuffer;
 import ddmd.root.rmem;
 import ddmd.root.rootobject;
+import ddmd.target;
```

Comment:
I've tried to look at the PR and commit history to find the fix, but was unable to find it.